guides/common/modules/ref_system-requirements.adoc

```diff
@@ -49,15 +49,21 @@ Before you install {ProductName}, ensure that your environment meets the require
 {ProductName} must be installed on a freshly provisioned system that serves no other function except to run {ProductName}.
 The freshly provisioned system must not have the following users provided by external identity providers to avoid conflicts with the local users that {ProductName} creates:
 
+ifdef::foreman-deb[]
+* www-data
+endif::[]
+ifndef::foreman-deb[]
 * apache
+endif::[]
+ifeval::["{context}" == "{project-context}"]
 * foreman
+endif::[]
 * foreman-proxy
 * postgres
 ifdef::katello,satellite[]
```

Comment:
@maximiliankolb shouldn't this (and qdrouterd below) also include orcharhino?

Reply:
Yes; if it's necessary for katello.